nepi/urls.py

```diff
@@ -142,6 +144,11 @@
 )
 
 
+urlpatterns += i18n_patterns(
```

Comment:
These flatpages are already defined within the admin console (see /admin/flatpages/flatpage/1/). What's the thought behind configuring urls here?